ofl/ibarrarealnova/METADATA.pb

```diff
@@ -0,0 +1,62 @@
+name: "Ibarra Real Nova"
+designer: "Jos\303\251 Mar\303\255a Ribagorda, Octavio Pardo"
```

Comment:
This is bad, looks like a bug in our tooling

Reply:
That's serialized by the python protocol buffers library, it would be interesting if the correct string is in the parsed result.

Reply:
Looks good to me:

```
In [1]: import gftools.fonts_public_pb2 as fonts_pb2
In [2]: from google.protobuf import text_format
In [3]: with open('METADATA.pb') as f: pbtxt = f.read()
In [4]: msg = fonts_pb2.FamilyProto()
In [5]: text_format.Merge(pbtxt, msg)
In [6]: print(msg.designer)
José María Ribagorda, Octavio Pardo
```

Reply:
I also checked on the metadata at that moment and thought the same. Is it ok?

Reply:
Issue is here: https://github.com/googlefonts/gftools/issues/161